templates/template_test.go

```diff
@@ -288,7 +288,7 @@ func mockupCommonvariables(t *testing.T) jet.VarMap {
 	variables.Set("User", &models.User{})
 	variables.Set("URL", &url.URL{})
 	variables.Set("CsrfToken", "xxxxxx")
-	variables.Set("EUCookieLaw", "test")
+	variables.Set("EUCookieLaw", "false")
```

Comment:
No I mean, don't use "false", but `false`

Reply:
i know it was just a mistake